modules/exploits/windows/http/rejetto_hfs_rce_cve_2024_23692.rb

```diff
@@ -0,0 +1,144 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+  prepend Msf::Exploit::Remote::AutoCheck
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Rejetto HTTP File Server (HFS) Unauthenticated Remote Code Execution',
+        'Description' => %q{
+          The Rejetto HTTP File Server (HFS) version 2.x is vulnerable to an unauthenticated server side template
+          injection (SSTI) vulnerability. A remote unauthenticated attacker can execute code with the privileges
+          of the user account running the HFS.exe server process. This exploit has been tested to work against version
+          2.4.0 RC7 and 2.3m. The Rejetto HTTP File Server (HFS) version 2.x is no longer supported by the maintainers
+          and no patch is available. Users are recommended to upgrade to version 3.x.
```

Comment:
```suggestion
          and no patch is available. Users are recommended to upgrade to newer supported versions.
```

since we don't want to bump this once `3.x` isn't supported anymore :)

Reply:
Good point, thanks @jvoisin 👍